app/jobs/create_uphold_cards_job.rb

```diff
@@ -2,24 +2,23 @@
 class CreateUpholdCardsJob < ApplicationJob
   queue_as :default
 
-  def perform(publisher_id:)
-    publisher = Publisher.find(publisher_id)
-
-    unless publisher.uphold_connection.can_create_uphold_cards?
-      Rails.logger.info("Could not create uphold card for publisher #{publisher.id}.")
-      SlackMessenger.new(message: "Could not create uphold card for publisher #{publisher.id}.", channel: SlackMessenger::ALERTS).perform
+  def perform(uphold_connection:)
+    unless uphold_connection.can_create_uphold_cards?
+      # Reasons might include that they are not in a district which allows for crypto, like tennesse
```

Comment:
I believe this particular error would not result in `can_create_uphold_cards` being false - the user will be able to create cards, just not ones denominated in crypto IIRC